src/reformatters/common/config_models.py

```diff
@@ -79,7 +79,7 @@ class Encoding(pydantic.BaseModel):
 
     # Could be any np.typing.DTypeLike but that type is loose and allows any string.
     # It's fine to add any valid dtype string to this literal.
-    dtype: Literal["float32", "float64", "uint16", "int64", "bool"]
```

Comment:
This should probably be int16 and we should figure out how to handle the mask. This int32 was left over from a previous attempt and needs to be changed.